vhdbuilder/packer/packer_source.sh

```diff
@@ -87,6 +87,70 @@ copyPackerFiles() {
   RECONCILE_PRIVATE_HOSTS_DEST=/opt/azure/containers/reconcilePrivateHosts.sh
   KUBELET_SERVICE_SRC=/home/packer/kubelet.service
   KUBELET_SERVICE_DEST=/etc/systemd/system/kubelet.service
+  
+  CSE_REDACT_SRC=/home/packer/cse_redact_cloud_config.py
```

Comment:
broke from the pattern because I felt this was easier to review/read, tried to keep all permissions as is except where I am sure I can reduce them like for *.service unit files.